### ISSUE TITLE
Show last update date of ELI below Layer list

### ISSUE THREAD

**Issue by tordans** [openstreetmap/iD#10238]
ATM, it is way to hard to understand how the pieces of ELI + iD play together.

One way to fix this are more regular updates to ELI like https://github.com/openstreetmap/iD/issues/9730 and https://github.com/openstreetmap/iD/issues/8466 suggest.

But maybe we can start with just showing the age of the data in the sidebar? So users understand why they don't see updates?

Something like
> Editor Layer Index last updated 2024-01-01

<img width="451" alt="Bildschirmfoto 2024-05-10 um 10 10 02" src="https://github.com/openstreetmap/iD/assets/111561/5b600622-0803-4086-aa42-6403ca736540">

**Comment by andrewharvey** [comment]
I think that's only really useful for someone digging into ELI and iD, for a regular user it doesn't really help to tell them the index was last updated. It would be better to just move ahead with #9730 / #8466

**Comment by tyrasd** [comment]
I would agree with @andrewharvey that it's probably most useful to always fetch a fresh version of ELI.
